tests/robot-tests/.env.example

```diff
@@ -19,5 +19,4 @@ TIMEOUT=30 # Variable used throughout tests (in seconds)
 IMPLICIT_WAIT=15 # Variable used throughout tests (in seconds)
 FAIL_TEST_SUITES_FAST=1 # ("0" or "1") Fails entire test suite after any failure
 IDENTITY_PROVIDER= # ("AZURE" or "KEYCLOAK") opt into using either a Keycloak or Azure based identity provider
-WAIT_MEMORY_CACHE_EXPIRY= # Time to wait for the memory cache to expire (in minutes. Local = 2, Dev = 10)
```

Comment:
Just wondering if this change is intentional?

Reply:
Yep it is, someone put the same key twice (see L14 above)